test/e2e-tests.sh

```diff
@@ -23,6 +23,11 @@ failed=0
 # Run serverless-operator specific tests.
 (( !failed )) && serverless_operator_e2e_tests || failed=2
 (( !failed )) && ensure_serverless_installed || failed=3
+if [[ $TEST_KNATIVE_KAFKA == true ]]; then
```

Comment:
How's this supposed to work with the test target we added? :thinking:

Reply:
@markusthoemmes 
we want to progress this as follows:
- see if things are green w/o KnativeKafka serverless e2e tests (aka deployment shape checks)
- once this PR is green, we merge https://github.com/openshift/release/pull/12939 and afterwards we start having a separate `e2e-test-all` target. This runs the deployment shape checks for KnativeKafka too
This way, the existing `make test-e2e` has the same behavior (when done in local) but we use `test-e2e-all` in CI in addition.

@matzew is now on fixing the wrong target name added on master: https://github.com/openshift-knative/serverless-operator/blob/master/Makefile#L40

Reply:
This should be good now. Prow is configured already